# Unique identifier
int32 vertex_id

# Physical location
geometry_msgs/Point location

# Information gain for goal reselection
float32 information_gain
